modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItFunctionsTest.java

```diff
@@ -344,6 +351,193 @@ public void testSubstr() {
         assertThrowsWithCause(() -> sql("SELECT SUBSTR('1234567', 1, -3)"), IgniteException.class, "negative substring length");
     }
 
+    /** Test cases for ROUND function that accepts integral numeric types. */
+    @TestFactory
+    public Stream<DynamicTest> testRoundIntegral() {
```

Comment:
please rewrite these tests without TestFactory. You don't need `TestFactory` until you are working on a test framework.

Reply:
Done.